core/modules/field_ui/field_ui.admin.inc

```diff
@@ -10,6 +10,9 @@
  * @see field_ui_menu()
  */
 function field_ui_manage_display_title($entity_type, $machine_name) {
+  if ($machine_name === 'default') {
+    return 'Default';
```

Comment:
This just needs to be wrapped in `t()`. The entity view mode labels within `entity_view_mode_load()` are already translated.